liferay-lima/liferay-portal
# LPS-168568 Generalize logic in dm display context

# What is this about?

https://issues.liferay.com/browse/LPS-168568

There's some unnecessary logic in display contexts to filter out system structures and hide some controls.

Infer that information from the file entry type or ddm structure and remove any custom logic.

# How do I test this?

There should be no changes in behavior. You can verify the following:

- Create a video shortcut or google docs shortcut and verify that the structure fileds are not displayed in the info panel, and that the video/document cannot be downloaded.
